images/base/files/etc/sysctl.d/10-network-magic.conf

```diff
@@ -0,0 +1,7 @@
+# Do not consider loopback addresses as martian source or destination while routing
+# Since pods mount the node resolv.conf, they can't use a loopack address that is only
+# reachable from the node, so KIND rewrites the well-known docker DNS address 127.0.0.11 
+# by a non-loopback address.
+# The DNS traffic coming from a pod will have to route to a localhost address to be NATed
+# hence we have to enable route_localnet or pods DNS will not work.
+net.ipv4.conf.all.route_localnet=1
```

Comment:
Seems to cause https://nvd.nist.gov/vuln/detail/CVE-2020-8558

Maybe fine for `kind` usecases, but I think at least the CVE should be explained here.

Reply:
heh, thanks for the comment
I forgot that kubelet install iptables rules to mitigate the issue, so this doesn't suppose a risk for kind
cc @BenTheElder

Reply:
I don't think this is fully mitigated https://github.com/kubernetes/kubernetes/issues/92315#issuecomment-656506304

Reply:
I disagree that is a real issue https://github.com/kubernetes/kubernetes/pull/91666#issuecomment-849416083